commands/azureCommands/delete-azure-image.ts

```diff
@@ -53,7 +53,13 @@ export async function deleteAzureImage(context?: AzureImageNode): Promise<void>
         tag = wholeName[1];
     }
 
-    let creds = await acrTools.loginCredentials(subscription, registry);
+    let creds;
+    try {
+        creds = await acrTools.loginCredentialsAccessToken(context.subscription, context.registry, context);
+    } catch (error) {
+        console.log(error);
```

Comment:
Is user able to see the log? If yes, should the code throw or prompt the error message?

Reply:
I'm removing the try catch statement. @estebanreyl  said that Stephen added something that gives users appropriate error messages now and the error would show in the terminal